.github/workflows/build-ci-develop.yml

```diff
@@ -0,0 +1,53 @@
+name: Build-CI-Develop
+on:
+  pull_request:
+    branches: [develop]
+  push:
+    branches: [develop]
+jobs:
+  build:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Rust Dependecies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      #       - name: Install toolchain
+      #         run: ./install_rust.sh
+      - name: Build Project
+        if: github.event_name == 'push'
+        run: cargo build --release
+      - run: cargo clean
+      - name: Test Project
+        run: cargo test
+      - name: Run Cargo Trampulin
+        if: github.event_name == 'push'
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.15.0'
+          args: '--exclude polkadex-node node-polkadex-runtime polkadex-primitives  --workspace'
+      - name: Upload to codecov.io
+        if: github.event_name == 'push'
+        uses: codecov/codecov-action@v1.0.2
+        with:
+          token: ${{secrets.CODECOV_TOKEN}}
+      - name: Archive code coverage results
+        if: github.event_name == 'push'
+        uses: actions/upload-artifact@v1
+        with:
+          name: code-coverage-report
+          path: cobertura.xml
+      - name: Set Node.js version 14.17.0
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.17.0
+      - name: Install packages
+        run: npm install
+      # - name: Runtime Upgrade
+      #   if: github.event_name == 'push'
```

Comment:
This should be added to main branch and here it should push to internal testnet

Reply:
Yes. I'm aware of that. I haven't yet established the internal testnet one. Can I create EC2 instances for that?

Reply:
Create a separate work flow for push and remove all `if event == push` there will be some duplication but it will be easier to read

Reply:
I don't agree. If so, it will duplicate the other stuff and be difficult to change. If it gives readability, I'm not sure why `if event == push` command is available on github actions.

Reply:
@simonsso I make them seperates

Reply:
@Gauthamastro, First we should run a few iteraterations on  develop -> internaltestnet.polkadex.trade upgrades when that is ok we will move on with main -> blockchain.plokadex.trade upgrades